utils/wasm-builder/src/lib.rs

```diff
@@ -39,6 +40,7 @@ pub const TARGET: &str = env!("TARGET");
 pub struct WasmBuilder {
     wasm_project: WasmProject,
     cargo: CargoCommand,
+    excluded_features: Vec<&'static str>,
```

Comment:
I'd rename it as `excluded` may confuse. Maybe `scoped_features`? With some description as a doc-comment.

Reply:
Imo, `scoped` says even less, and frunkly nothing to me. I can suggest `omitted`, `ignored`. Any other preferences?

Reply:
Maybe `skipped` but it is pretty the same as `excluded` :)